### ISSUE TITLE
Struggling with Firefox Browser.

### ISSUE THREAD

**Issue by brunano21** [SeleniumHQ/selenium#1053]
Hello,
I'm writing some tests using intern. I'm not sure if this is an intern issue or a selenium one. So I posted in both sides. :-1: 
The same tests run well against Google Chrome, by the way.
This is my configuration:

Firefox ESR v 38.2.1
Windows 7 x64
Selenium Standalone server 2.47.1
Intern v3.0.3

Intern.conf.env: environments: [{ browserName: 'firefox', version: '38', platform: 'WINDOWS'}]

This is the output of selenium server: http://pastebin.com/duDcGYp1
Basically, the browser starts, does stuff, closes by itself, then my test fail, and finally a new browser windows has been opened.

Can anyone suggest any solution? Or am I missing something?
Thanks.

**Comment by smilinrobin** [comment]
Sir , I see multiple issues in the output of Selenium server:
1. 10:56:13.567 INFO - Driver class not found: com.opera.core.systems.OperaDriver
Is the Opera driver properly setup/downloaded/instantiated?
2. Also, poking my nose around the internet , I found a similar thread on StackOverflow at : http://stackoverflow.com/questions/25393684/intern-2-0-fails-with-local-selenium-grid

**Comment by brunano21** [comment]
Hello, thanks for replying.
1. I always have that kind of `warning` every time I launch selenium. But I do not care because I'm not using OperaDriver.
2. I'm using Intern v3.0, instead that thread is referring to Intern v2.0.

What it's not clear is why FF closes itself and then gets opened again. Moreover, Selenium logs does not say anything about this.

**Comment by lukeis** [comment]
@brunano21 please provide a sample test case to reproduce your issue. Otherwise support like this is best done on the selenium-users mailing list:
https://groups.google.com/forum/#!forum/selenium-users

going to close for now until something reproducible is provided.
